tests/unit/header-cell-test.js

```diff
@@ -0,0 +1,42 @@
+import Ember from 'ember';
+import {
+  moduleFor,
+  test
+} from 'ember-qunit';
+
+import ColumnDefinition from '../../models/column-definition';
+
+moduleFor('view:headerCell', 'Unit: Header Cell', {
+  unit: true,
+  needs: [
+    'template:header-cell'
+  ]
+});
+
+test('The elementSizeDidChange scheduled task is correctly canceled when the element is destroyed', function(assert) {
+  assert.expect(3);
+
+  let headerCell = this.subject({
+    content: ColumnDefinition.create({}),
+    tableComponent: {
+      columnMode: 'standard'
+    }
+  });
+
+  Ember.run(function() {
```

Comment:
`this.append` or `this.render` ( I forgot which version of Ember QUnit is used ). No need to wrap it in a run loop, no?